subprojects/docs/src/docs/userguide/buildLifecycle.adoc

```diff
@@ -79,6 +79,7 @@ In the settings file you can use a set of methods to build the project tree. Hie
 ++++
 
 The `include` method takes project paths as arguments. The project path is assumed to be equal to the relative physical file system path. For example, a path 'services:api' is mapped by default to a folder 'services/api' (relative from the project root). You only need to specify the leaves of the tree. This means that the inclusion of the path 'services:hotels:api' will result in creating 3 projects: 'services', 'services:hotels' and 'services:hotels:api'.
+Note that the project path delimiter is `:` and **not** `/`. If your project is located in a physical location that is different to the project path, you can set the `projectDir` property. For example, `projects.services.projectDir = 'subprojects/services'`.
```

Comment:
❌ We should explain this properly in the user guide and just link it from here as well as form the deprecation warning. The user guide should show examples of the most common configurations:

- including a project with a hierarchical path
- including a project from a subfolder, but not creating a logical hierarchy
- including several projects by iterating over the contents of a folder

Reply:
We say that you should use `:` in the [DSL reference](https://docs.gradle.org/nightly/dsl/org.gradle.api.initialization.Settings.html#org.gradle.api.initialization.Settings:include(java.lang.String[])), but `/` "works" so people don't dig any deeper.

+1 on adding examples. I think longer examples should go into the class Javadoc for Settings so it shows up in our DSL reference.

Reply:
Good point, having it in the JavaDoc is even better, e.g. for Kotlin.